githubcontent/main.py

```diff
@@ -0,0 +1,89 @@
+from bs4 import BeautifulSoup
+import json
+import re
+import requests
+
+def get_github_urls(base_url: str, chapters: list) -> list:
+    """
+    get a list of urls 
+    """
+    urls = []
+    for chapter in chapters:
+        urls.append(base_url.format(chapter))
+        
+    print(urls[-5:])
+    return urls
+
+def parse_chapters(urls):
+
+    documents = []
+    for url in urls:
+        is_bitcoin_url = re.search('bitcoinbook', url)
+        data = requests.get(url).text
+        soup = BeautifulSoup(data,'html.parser')
+        document = {}
+        title = soup.find('h2', dir='auto').get_text()
+        body = soup.find('div',id = 'readme').get_text()
+        body_type = "md"
+        author = "Andreas Antonopoulos"
+        chapter_number = ''.join(re.findall(r'\d+', url))
+        id = "bitcoinbook-chapter-" + chapter_number if is_bitcoin_url else "lnbook-chapter-" + chapter_number 
+        domain = "https://github.com"
+        url = url
+        created_at = "2022-11-15" if is_bitcoin_url else "2023-04-22"# date of most recent commit
+
+        document.update({
+            "title": title,
+            "body": body,
```

Comment:
I'd like to add a field that is body_formatted as well to reflect https://github.com/bitcoinsearch/scraper/issues/18

Reply:
@adamjonas should this be a boolean or some kind of enumeration?

Reply:
no, this is likely a formatted body similar to how things are parsed in our markdown parsers:

https://github.com/bitcoinsearch/scraper/blob/master/bitcoinops/main.js#L94